sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/rx/ReadFeedUsersTest.java

```diff
@@ -37,11 +37,9 @@ public ReadFeedUsersTest(CosmosClientBuilder clientBuilder) {
 
     @Test(groups = { "simple" }, timeOut = FEED_TIMEOUT)
     public void readUsers() throws Exception {
-
-        FeedOptions options = new FeedOptions();
         int maxItemCount = 2;
 
-        CosmosPagedFlux<CosmosUserProperties> feedObservable = createdDatabase.readAllUsers(options);
+        CosmosPagedFlux<CosmosUserProperties> feedObservable = createdDatabase.readAllUsers();
 
         int expectedPageSize = (createdUsers.size() + maxItemCount - 1) / maxItemCount;
```

Comment:
result validation validates page size, etc. that may fail. you need to update the validation section.

Reply:
Page sizes are controlled by maxItemCount which is no longer part of the FeedOptions since when we start returning PageFlux as results.